ecoscope/analysis/seasons.py

```diff
@@ -110,21 +113,21 @@ def seasonal_windows(ndvi_vals, cuts, season_labels):
 
 
 def add_seasonal_index(
-    df, index_name, start_date, end_date, aoi_geom_filter=None, seasons=2, season_labels=["dry", "wet"]
+    df, index_name, start_date, end_date, time_col, aoi_geom_filter=None, seasons=2, season_labels=["dry", "wet"]
 ):
 
     aoi_ = None
     try:
         aoi_ = aoi_geom_filter.dissolve().iloc[0]["geometry"]
```

Comment:
here we should do the unary_union of all geometries passed in: https://shapely.readthedocs.io/en/stable/reference/shapely.unary_union.html

Reply:
Just to be sure, you mean in what's provided to `aoi_geom_filter`?

I should have called out the changes to `add_seasonal_index` were just to get our linter to allow the commit. It's not covered by any tests currently.

Reply:
Ok, ignore my comment for now and we can revisit if needed. Thanks